packages/backend/src/open_payments/payment/outgoing/lifecycle.ts

```diff
@@ -156,9 +156,12 @@ export async function handleSending(
     )
     newMinDeliveryAmount = payment.receiveAmount.amount - amountDelivered
   } else {
-    if (!destination.invoice) throw LifecycleError.MissingIncomingPayment
+    if (!destination.destinationPaymentDetails)
+      throw LifecycleError.MissingIncomingPayment
     newMinDeliveryAmount =
-      destination.invoice.amountToDeliver - destination.invoice.amountDelivered
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      destination.destinationPaymentDetails.incomingAmount!.amount -
```

Comment:
Since it's possible for `incomingAmount` to be undefined, maybe the `if (payment.receivingAccount)` above should be changed to:
```
if (payment.receivingAccount || !destination.destinationPaymentDetails?.incomingAmount)
```